packages/react-next/src/components/Keytip/examples/Keytips.Basic.Example.tsx

```diff
@@ -6,7 +6,7 @@ import { Dropdown } from '@fluentui/react-next/lib/Dropdown';
 import { Link } from '@fluentui/react-next/lib/Link';
 import { SpinButton, ISpinButtonStyles } from '@fluentui/react-next/lib/SpinButton';
 import { Toggle } from '@fluentui/react-next/lib/Toggle';
-import { Pivot, PivotItem } from '@fluentui/react-next/lib/Pivot';
+import { Pivot, PivotItem } from '@fluentui/react-tabs';
```

Comment:
Since you are already re-exporting from `react-tabs` in `Pivot.ts` you can probably leave this unchanged.